pkg/kubelet/dockershim/docker_container.go

```
@@ -243,10 +239,12 @@ func (ds *dockerService) createContainerLogSymlink(containerID string) error {
 			glog.V(10).Infof("Docker logging driver is %s", dockerLoggingDriver)
 		}
 
-		if dockerLoggingDriver == dockerDefaultLoggingDriver {
-			glog.Errorf("Cannot create symbolic link because container log file doesn't exist!")
-		} else {
-			glog.V(5).Infof("Unsupported logging driver: %s", dockerLoggingDriver)
+		for _, driver := range criSupportedLogDrivers {
+			if dockerLoggingDriver == driver {
+				glog.Errorf("Cannot create symbolic link because container log file doesn't exist!")
```

shouldn't this return an error instead of logging it?